scripts/pptx_extract.py

```diff
@@ -0,0 +1,43 @@
+from pptx import Presentation
+
+prs = Presentation("files/ppt_demo.pptx")
+
+# text_runs will be populated with a list of strings,
+# one for each text run in presentation
+#text_runs = []
+
+
+# slide_num = len(prs.slides)  # to calculate slides number
+slide_text = []
+slide_text = ['---','\n','layout: presentation','\n','author: '+ prs.core_properties.author,'\n','title: ' + prs.core_properties.title]
```

Comment:
It might be more readable to break down each of these elements on a separate line. For example:
```python
author = prs.core_properties.author
slide_text = [..., author, ...]
```